asteroid/filterbanks/multiphase_gammatone_filterbank.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import torch
+from .enc_dec import Filterbank
+
+class MultiphaseGammatoneFB(Filterbank):
+    """ Multi-Phase Gammatone Filterbank as described in [1].
+        Please cite [1] whenever using this.
+        Original code repository: `<https://github.com/sp-uhh/mp-gtf>`
+
+    References:
+    [1] David Ditter, Timo Gerkmann, "A Multi-Phase Gammatone Filterbank for Speech
+        Separation via TasNet", IEEE Int. Conf. Acoust., Speech, Signal Process. (ICASSP),
+        Barcelona, Spain, May 2020.
+        Available: `<https://ieeexplore.ieee.org/document/9053602/>`
+
+    Args:
+        n_filters (int): Number of filters. 
+        kernel_size (int): Length of the filters.
+        sample_rate (int, optional): The sample rate (used for initialization).
+        stride (int, optional): Stride of the convolution. If None (default),
+            set to ``kernel_size // 2``.
+    """
+    
+    def __init__(self, n_filters=128, kernel_size=16, sample_rate=8000, stride=None):
+        super(MultiphaseGammatoneFB, self).__init__(n_filters, kernel_size, stride=stride)
+        self.sample_rate = sample_rate
+        self.n_filters = n_filters
+        self.kernel_size = kernel_size
+
+    @property
+    def filters(self):
+        """ Compute filters from parameters """
+        length_in_seconds = self.kernel_size / self.sample_rate
+        filterbank = generate_mpgtf(self.sample_rate, length_in_seconds, self.n_filters)
+        filterbank = np.expand_dims(filterbank, axis=1)
+        return torch.from_numpy(filterbank).float()
+
+### UTILITY FUNCTIONS ###
```

Comment:
The next section of the code doesn't comply with our coding style. 
You can use a linter to reformat it accordingly, or [`black`](https://github.com/psf/black) with line length of 79 (`black -l 79 file.py`)

Reply:
Ok, I used `black -l 79 file.p`. Output looks a little bit weird to me. Let me know if this is fine.

Reply:
Yes, this is fine, thanks! :wink: